Worksheets/worksheet_01.txt

```diff
@@ -79,67 +92,98 @@
      area = pi  * radius ** 2
      print(area)
      
+     There are several uneeded steps.
+     
  11. Explain the mistake in the following code:
      
      x = 4
      y = 5
      a = ((x) * (y))
      print(a)
      
+     The variables didnt need to be in ()
+     
  12. Explain the mistake in the following code:
      
      x = 4
      y = 5
      a = 3(x + y)
      print(a)
      
+      The 3 isnt defined and the program doesnt know what to do with the 3. the program doesnt know to apply the distrubitive preoperty.
+     
+     
  13. Explain the mistake in the following code:
      
      radius = input(float("Enter the radius:"))
      
+     There is no command to print. so it takes what you say and does the things but doesnt ever show back up.
+     
  14. Do all these print the same value? Which one is better to use and why?
      
      print(2/3+4)
      print(2 / 3 + 4)
-     print(   2 /    3+    4  )
+     print(   2 /    3+    4  )]
+     
+     Only the first one will work or it will give you a indentation error. 
      
  15. What is a constant?
 
+ It is a varibale that is set in stone.
+ 
  16. How are variable names for constants different than other variable names?
 
+ They are camel-cased. (ex: " helloTherePeople")
+ 
  17. What is a single quote and what is a double quote?
      Give and label an example of both.
 
+    'a' - single quote
+    "A" - double quotes
+     
  18. Write a Python program that will use escape codes to print a double-quote
      and a new line using the Window's standard. (Note: I'm asking for the Window's
      standard here. Look it up out of Chapter 1.)
 
+     print("Hello."\n)
+     
  19. Can a Python program print text to the screen using single quotes instead
      of double quotes?
 
+     Yes as long as you stick with whatever quotes you use it will be fine.
+     
  20. Why does this code not calculate the average?
      
      print(3 + 4 + 5 / 3)
      
+     Because it isnt instructed to. You would need another set of ()
 

```

Comment:
try for a more detailed explanation.  WHY isn't it instructed to?  what is wrong with the statement that prevents an average from being calulated?